packages/resp/resp.0.10.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "Zach Shipko <zachshipko@gmail.com>"
+authors: ["Zach Shipko <zachshipko@gmail.com>"]
+homepage: "https://github.com/zshipko/resp"
+doc: "https://zshipko.github.io/resp"
+license: "ISC"
+dev-repo: "git+https://github.com/zshipko/resp.git"
+bug-reports: "https://github.com/zshipko/resp/issues"
+depends:
+[
+    "ocaml" {>= "4.07.0"}
+    "dune" {build & >= "2.0.0"}
```

Comment:
```suggestion
    "dune" {>= "2.0.0"}
```

Reply:
Was the build flag also in the previous releases? If so,  that needs to be fixed (not necessarily here, can be a separate PR)

Reply:
The flag was removed already in the previous releases